engine/runtime/src/main/java/org/enso/interpreter/runtime/Module.java

```diff
@@ -193,19 +197,40 @@ public Rope getLiteralSource() {
    * @param source the module source.
    */
   public void setLiteralSource(String source) {
-    setLiteralSource(Rope.apply(source));
+    setLiteralSource(Rope.apply(source), null);
   }
 
   /**
    * Sets new literal sources for the module.
    *
    * @param source the module source.
+   * @param update suggested small change in a single literal or {@code null}
```

Comment:
Please describe in the comment the behavior if `update` is not null. It's not clear from the method name.

Reply:
fd6702509